### PR TITLE
combines filter chains into a single node

### PR DESCRIPTION
**Pull request by larat7**
**THIS**
image

**INSTEAD OF** 

image

### CODE REVIEW

File: src/sql/mir.rs [hunk 1]
```diff
@@ -60,13 +60,15 @@ impl SqlToMirConverter {
         SqlToMirConverter { log: log, ..Default::default() }
     }
 
-    /// Converts a condition tree stored in the `ConditionExpr` returned by the SQL parser into a
-    /// vector of conditions that `shortcut` understands.
-    fn to_conditions(&self,
+    /// Converts a condition tree stored in the `ConditionExpr` returned by the SQL parser
+    /// and adds its to a vector of conditions that `shortcut` understands.
```
Comment (jonhoo) [inline src/sql/mir.rs]: Just btw, we don't use `shortcut` any more :p

**Comment by jonhoo** [comment]
LGTM, but r? @ms705

File: src/sql/mir.rs (hunk 1 above)
Comment (ms705) [inline src/sql/mir.rs]: That's on me :) I wrote that comment and didn't update it.

**Comment by ms705** [comment]
Ok -- I think there are some possible improvements, but it's correct and in the interest of expedience and other merges, we should merge this. Thanks @larat7!

**Pull request merged**